main.c

```diff
@@ -30,10 +31,7 @@ static int nsenter(pid_t target_pid)
 		perror(netns);
 		return netnsfd;
 	}
-	if (setns(usernsfd, CLONE_NEWUSER) < 0) {
-		perror("setns(CLONE_NEWUSER)");
-		return -1;
-	}
+	setns(usernsfd, CLONE_NEWUSER);
```

Comment:
why?

Reply:
in podman we already joined the userns, and attempting to join it again will fail.  I think it is fine to ignore errors here, as the join network namespace would still fail later on.  What do you think?

Reply:
I think we should fail here unless already joined, but can be amother pr